docs/guides/cloud/github-integration.mdx

```diff
@@ -268,6 +278,13 @@ them into Cypress Cloud, and complete the activation process.
    install the app against all repos or select specific ones, then click the
    **Install** button.
 
+:::info
```

Comment:
Same here with using the markdown text styling syntax

Reply:
This was on the wrong PR. It's been removed and will be adjusted on the correct PR

Reply:
Was this the correct comment here? I still see the note related to my first comment. If we're keeping it, it can match the other instance of this on the page using the markdown syntax.